docs/get-started/getting-started-gcp.md

```diff
@@ -74,9 +83,52 @@ gcloud dataproc clusters create $CLUSTER_NAME  \
     --optional-components=JUPYTER,ZEPPELIN \
     --metadata rapids-runtime=SPARK \
     --bucket $GCS_BUCKET \
+    --enable-component-gateway
+```
+
+This may take around 10-15 minutes to complete.  You can navigate to the Dataproc clusters tab in the
+Google Cloud Console to see the progress.
+
+![Dataproc Cluster](../img/GCP/dataproc-cluster.png)
+
+If you'd like to further accelerate init time to 4-5 minutes, create a custom Dataproc image using
+[this](#build-custom-dataproc-image-to-accelerate-cluster-init-time) guide.
+
+### Create a Dataproc Cluster using MIG with A100's
+* One 16-core master node and 5 12-core worker nodes
+* 1 NVIDIA A100 for each worker node, split into 2 MIG instances using
+[instance profile 3g.20gb](https://docs.nvidia.com/datacenter/tesla/mig-user-guide/#a100-profiles).
+
+```bash
+    export REGION=[Your Preferred GCP Region]
+    export GCS_BUCKET=[Your GCS Bucket]
+    export CLUSTER_NAME=[Your Cluster Name]
+    export NUM_GPUS=1
+    export NUM_WORKERS=5
+
+gcloud dataproc clusters create $CLUSTER_NAME  \
+    --region $REGION \
+    --image-version=2.0-ubuntu18 \
+    --master-machine-type n1-standard-16 \
+    --num-workers $NUM_WORKERS \
+    --worker-accelerator type=nvidia-tesla-a100,count=$NUM_GPUS \
+    --worker-machine-type a2-highgpu-1g \
+    --num-worker-local-ssds 4 \
+    --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/gpu/install_gpu_driver.sh,gs://goog-dataproc-initialization-actions-${REGION}/rapids/rapids.sh \
```

Comment:
a nit: it is a history issue but how about we change all `--parameter value` to `--parameter=value` which follows the https://cloud.google.com/sdk/gcloud/reference/dataproc/clusters/create?
So that all parameters of this command are in sync.

Reply:
updated